perfkitbenchmarker/linux_benchmarks/cassandra_stress_benchmark.py

```diff
@@ -355,9 +355,13 @@ def RunTestOnLoader(vm, loader_index, operations_per_vm, data_node_ips,
     population_dist: string. The population distribution.
     population_params: string. Representing additional population parameters.
   """
+
+  command = FLAGS.cassandra_stress_command
```

Comment:
Whats the reason for this? It looks like "command" gets passed in as an argument to this function